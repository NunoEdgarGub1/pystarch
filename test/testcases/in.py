x01 = 1 in [1, 2, 3]
x02 = 'a' in [1, 2, 3]
x03 = 'a' in {'a': 1, 'b': 2}
x04 = 1 in (1, 2, 3)
x05 = 1 in 2 in 3
x06 = 1 in 2
