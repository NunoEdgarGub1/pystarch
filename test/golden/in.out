x01 Bool
x02 Bool
x03 Bool
x04 Bool
x05 Bool
x06 Bool

testcases/in.py:2 inconsitent-types "In" (Str, Num)
testcases/in.py:4 in-operator-argument-not-list-or-dict "In"
testcases/in.py:5 in-operator-chaining "In In"
testcases/in.py:6 in-operator-argument-not-list-or-dict "In"
